File: p07_binary_right_angle_triangle/main.ts
const input = require("prompt-sync")();

let n: number = Number(input())

for(let i = 1; i <= n; i++){
    let row: string = "";
    for(let j = 1; j <= i; j++){
        if((i + j) % 2 == 0){
            row += "1 ";
        }else{
            row += "0 ";
        }
    }
    console.log(row);
}

File: p07_binary_right_angle_triangle/main.cpp
#include<iostream>
#include<string>

using namespace std;

int main(){
    int n;
    cin >> n;

    for(int i = 1; i <= n; i++){
        string row = "";

        for(int j = 1; j <= i; j++){
            if((i + j) % 2 == 0){
                row += "1 ";
            }else{
                row += "0 ";
            }
        }

        cout << row << endl;
    }

    return 0;
}